tests/pool_test.go

```diff
@@ -114,13 +114,14 @@ var _ = Describe("[sig-compute]VirtualMachinePool", func() {
 
 	newVirtualMachinePool := func() *poolv1.VirtualMachinePool {
 		By("Create a new VirtualMachinePool")
-		template := tests.NewRandomVMIWithEphemeralDiskAndUserdata(cd.ContainerDiskFor(cd.ContainerDiskCirros), "#!/bin/bash\necho 'hello'\n")
-		return newVirtualMachinePoolWithTemplate(template, true)
+		running := true
+		return createVirtualMachinePool(tests.NewRandomPoolFromVMI(libvmi.NewCirros(), int32(0), running))
```

Comment:
What does `int32(0)` means? Either we define variables for both parameters or none.

Reply:
I see your point - my hatred to boolean args has blinded me from seeing `replicas`'s opacity.

I can add; and I may also use this opportunity to move `NewRandomPoolFromVMI` out of the utils.go "junkyard" and next to its only user (this module). What do you say?

Reply:
I think that would be great!